### ISSUE TITLE
By pass private key input

### ISSUE THREAD

**Issue by CoderZhi** [iotexproject/iotex-explorer#762]
Given an account address, most of the information are public available. Thus, it is not a big problem to store the account address and show account details before user input private key. Also, private key is very sensitive, and it may scare user away.

Based on the above assumption, we shouldn't ask users to input private key upon web or desktop wallet opened. Users can choose to input their private key or public key or account address.
When users input their private keys (not from ledger), they can opt-in to store the private key for future usage.

In conclusion, we should do the following things:
1. For the first time users, they can choose to input Private Key or Public Key or account address
2. For return users, open the wallet with last account by default
3. Ask for private key on demand
4. User can choose to keep private key in memory for a while (say 10 minutes)
5. Store multiple accounts

@guo @raullenchai
